synapse/lib/grammar.py

```diff
@@ -423,14 +423,22 @@ def parse_cmd_string(text, off):
     valu, newoff = CmdStringer().transform(tree)
     return valu, off + newoff
 
+ESC_RE = regex.compile(r'\\(.)')
+
+def unescape(valu):
```

Comment:
This doesn't properly handle whitespace encodings such as `\t`

Reply:
replacing this with ast.literal_eval